### DISCUSSION TITLE
automatic/unattended start Update routine for ansible workflow

### DISCUSSION THREAD

**Discussion by stabmaster1985** [tteck/Proxmox#2652] [Help Needed] [answered]
Hello,
i want to update my paperless-ngx container (bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx.sh)")
But it is possible to start the update skript without the question "if i want to proceed" and the next step to select/start the update process?

Background: i want to integrate it in my ansible workflow (with backup/snapshot), but the interface question didn't work in ansible (i think).

Could you help me?

Best regards,
Frank

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/discussions/2522#discussioncomment-8597879
